test/setup/build.ts

```diff
@@ -46,7 +45,7 @@ async function build() {
     await zip.extract(null, tmpTestDir);
     await zip.close();
   } catch (error) {
-    throw new OutputAndExitError(error);
+    OutputAndExitError(error);
```

Comment:
Very confusing that this is upper case as it looks like an error constructor (but it is just a function).
Please change naming to make this easier to understand, i.e. `printErrorAndExit`.